obs-packaging/gen_versions_txt.sh

```diff
@@ -148,13 +150,57 @@ main() {
 			use_head=1
 			shift
 			;;
+		--tag)
+			use_tag=1
+			shift
+			;;
 		-*)
 			die "Invalid option: ${1:-}" "1"
 			shift
 			;;
 	esac
-	local branch="${1:-}"
-	[ -n "${branch}" ] || die "No branch specified" "1"
+
+	local kata_version=
+	if [ -n "$use_tag" ]; then
+		if [ -n "${use_head}" ]; then
+		       die "tag and head options are mutually exclusive"
+		fi
+
+		# We are generating versions based on the provided tag
+		local tag="${1:-}"
+		[ -n "${tag}" ] || die "No tag specified" "1"
+
+		# use the runtime's repository to determine branch information
+		local repo="github.com/kata-containers/runtime"
+		local repo_dir="runtime"
+		git clone --quiet "https://${repo}.git" "${repo_dir}"
+		pushd "${repo_dir}" >> /dev/null
```

Comment:
Are you trying to discard `stdout` and `stderr`? If so, this won't work - you need something like:

```sh
pushd "${repo_dir}" &>/dev/null
```

Same for the `popd`.